content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -283,9 +298,18 @@ set -ex
 psql file.sql
 ```
 
+ 
+</tab> 
+
+ 
+</TabbedContent>
+
+
 ## Integrated seeding with Prisma Migrate
 
-Prisma Migrate integrates seamlessly with your seeds, assuming you follow the steps in the previous section. When Prisma Migrate resets the development database, seeding is triggered automatically. Prisma Migrate resets the database and triggers seeding in the following scenarios:
+Prisma Migrate integrates seamlessly with your seeds, assuming you follow the steps in the previous section. When Prisma Migrate resets the development database, seeding is triggered automatically. When Prisma Migrate resets the development database, seeding is triggered automatically. 
+
+Prisma Migrate resets the database and triggers seeding in the following scenarios:
 
 - You manually run the `prisma migrate reset` CLI command.
-- The database is reset interactively in the context of using `prisma migrate dev` - for example, as a result of migration history conflicts or database drift.
+- The database is reset interactively in the context of using `prisma migrate dev` - for example, as a result of migration history conflicts or database schema drift.
```

Comment:
For both `reset` and `dev`, there is a `--skip-seed` CLI flag to skip seeding, I think it's worth quickly mentioning here. Maybe something like:

> When you want to use `prisma migrate dev` or `prisma migrate reset` without seeding, you can pass the `--skip-seed` flag.